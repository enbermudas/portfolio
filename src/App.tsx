import Window from "./components/Window";

const App = () => {
  return (
    <div className="App">
      App
    </div>
  );
}

export default App;
