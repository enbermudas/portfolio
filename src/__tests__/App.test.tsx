import { render } from "@testing-library/react";
import App from "../App";
import renderer from "../test-renderer";

describe("App", () => {
  it("Renders correctly", () => {
    const container = render(renderer(<App />));
    expect(container).toMatchSnapshot();
  });
});
