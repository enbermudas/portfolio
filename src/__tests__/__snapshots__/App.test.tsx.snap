// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="sc-bcXHqe gGMGtT"
          data-testid="desktop-testid"
        >
          <div
            class="sc-gswNZR iMExeb"
          >
            <button
              class="sc-bBABsx jDAFfy"
              data-testid="projects-testid"
            >
              <img
                alt="Proyectos"
                class="sc-iveFHk cQcgqu"
                src="projects.png"
              />
              <div
                class="sc-cwSeag QuwNU"
              >
                Proyectos
              </div>
            </button>
            <button
              class="sc-bBABsx jDAFfy"
              data-testid="experience-testid"
            >
              <img
                alt="Experiencia"
                class="sc-iveFHk cQcgqu"
                src="experience.png"
              />
              <div
                class="sc-cwSeag QuwNU"
              >
                Experiencia
              </div>
            </button>
            <button
              class="sc-bBABsx jDAFfy"
              data-testid="technologies-testid"
            >
              <img
                alt="Tecnologías"
                class="sc-iveFHk cQcgqu"
                src="technologies.png"
              />
              <div
                class="sc-cwSeag QuwNU"
              >
                Tecnologías
              </div>
            </button>
            <button
              class="sc-bBABsx jDAFfy"
              data-testid="education-testid"
            >
              <img
                alt="Educación"
                class="sc-iveFHk cQcgqu"
                src="education.png"
              />
              <div
                class="sc-cwSeag QuwNU"
              >
                Educación
              </div>
            </button>
            <button
              class="sc-bBABsx jDAFfy"
              data-testid="contact-testid"
            >
              <img
                alt="Contacto"
                class="sc-iveFHk cQcgqu"
                src="contact.png"
              />
              <div
                class="sc-cwSeag QuwNU"
              >
                Contacto
              </div>
            </button>
            <button
              class="sc-bBABsx jDAFfy"
              data-testid="send-email-testid"
            >
              <img
                alt="Enviar un correo"
                class="sc-iveFHk cQcgqu"
                src="send_email.png"
              />
              <div
                class="sc-cwSeag QuwNU"
              >
                Enviar un correo
              </div>
            </button>
            <button
              class="sc-bBABsx jDAFfy"
              data-testid="bin-testid"
            >
              <img
                alt="Papelera"
                class="sc-iveFHk cQcgqu"
                src="bin.png"
              />
              <div
                class="sc-cwSeag QuwNU"
              >
                Papelera
              </div>
            </button>
          </div>
          <div
            class="sc-dkrFOg flVGBD"
          >
            <div
              class="sc-eDvSVe doLnMB"
            >
              <button
                class="sc-jSUZER kfTyWZ"
                data-testid="start-button-testid"
              >
                <img
                  alt="start-icon"
                  class="sc-gKPRtg kvlfLI"
                  src="start.png"
                />
                Start
              </button>
              <div
                class="sc-hLBbgP kktPrb"
              />
              <div
                class="sc-iBYQkv kwraTM"
              />
            </div>
            <div
              class="sc-jrcTuL jzpkNd"
            >
              <img
                alt="calendar_icon"
                class="sc-kDvujY eyeZVy"
                src="calendar_mini.png"
              />
              <img
                alt="speaker_icon"
                class="sc-kDvujY eyeZVy"
                src="speaker_mini.png"
              />
              <div
                data-testid="clock-testid"
              >
                13:27
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="sc-bcXHqe gGMGtT"
        data-testid="desktop-testid"
      >
        <div
          class="sc-gswNZR iMExeb"
        >
          <button
            class="sc-bBABsx jDAFfy"
            data-testid="projects-testid"
          >
            <img
              alt="Proyectos"
              class="sc-iveFHk cQcgqu"
              src="projects.png"
            />
            <div
              class="sc-cwSeag QuwNU"
            >
              Proyectos
            </div>
          </button>
          <button
            class="sc-bBABsx jDAFfy"
            data-testid="experience-testid"
          >
            <img
              alt="Experiencia"
              class="sc-iveFHk cQcgqu"
              src="experience.png"
            />
            <div
              class="sc-cwSeag QuwNU"
            >
              Experiencia
            </div>
          </button>
          <button
            class="sc-bBABsx jDAFfy"
            data-testid="technologies-testid"
          >
            <img
              alt="Tecnologías"
              class="sc-iveFHk cQcgqu"
              src="technologies.png"
            />
            <div
              class="sc-cwSeag QuwNU"
            >
              Tecnologías
            </div>
          </button>
          <button
            class="sc-bBABsx jDAFfy"
            data-testid="education-testid"
          >
            <img
              alt="Educación"
              class="sc-iveFHk cQcgqu"
              src="education.png"
            />
            <div
              class="sc-cwSeag QuwNU"
            >
              Educación
            </div>
          </button>
          <button
            class="sc-bBABsx jDAFfy"
            data-testid="contact-testid"
          >
            <img
              alt="Contacto"
              class="sc-iveFHk cQcgqu"
              src="contact.png"
            />
            <div
              class="sc-cwSeag QuwNU"
            >
              Contacto
            </div>
          </button>
          <button
            class="sc-bBABsx jDAFfy"
            data-testid="send-email-testid"
          >
            <img
              alt="Enviar un correo"
              class="sc-iveFHk cQcgqu"
              src="send_email.png"
            />
            <div
              class="sc-cwSeag QuwNU"
            >
              Enviar un correo
            </div>
          </button>
          <button
            class="sc-bBABsx jDAFfy"
            data-testid="bin-testid"
          >
            <img
              alt="Papelera"
              class="sc-iveFHk cQcgqu"
              src="bin.png"
            />
            <div
              class="sc-cwSeag QuwNU"
            >
              Papelera
            </div>
          </button>
        </div>
        <div
          class="sc-dkrFOg flVGBD"
        >
          <div
            class="sc-eDvSVe doLnMB"
          >
            <button
              class="sc-jSUZER kfTyWZ"
              data-testid="start-button-testid"
            >
              <img
                alt="start-icon"
                class="sc-gKPRtg kvlfLI"
                src="start.png"
              />
              Start
            </button>
            <div
              class="sc-hLBbgP kktPrb"
            />
            <div
              class="sc-iBYQkv kwraTM"
            />
          </div>
          <div
            class="sc-jrcTuL jzpkNd"
          >
            <img
              alt="calendar_icon"
              class="sc-kDvujY eyeZVy"
              src="calendar_mini.png"
            />
            <img
              alt="speaker_icon"
              class="sc-kDvujY eyeZVy"
              src="speaker_mini.png"
            />
            <div
              data-testid="clock-testid"
            >
              13:27
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
